builder/builder.go

```diff
@@ -608,17 +648,40 @@ func getBundleSetKeysSorted(set bundleSet) []string {
 	return keys
 }
 
+// isLocalPackagePath checks if path is a local-packages definition file
+func isLocalPackagePath(path string) bool {
+	return strings.HasSuffix(path, "/local-packages")
```

Comment:
I'd argue that this should be configurable in `builder.conf` alongside `LOCAL_BUNDLE_DIR`, presumably as `LOCAL_PACKAGE_LIST` or the like, and parsed into `b.LocalPackagesFile` (to mirror `b.MixBundlesFile`).

Reply:
I'm hesitant to make configuration changes with #237 still outstanding...

Reply:
Fair point. But if you don't do it not, we'll just have to do it later. People loves them their puttin' stuff outside the workdir.

Reply:
err, if I do it now, #237 will have to change OR I will have to redo the changes after #237 is merged. I'd rather make an issue to move this to a config. I'll even assign it to myself if it'll make you happy.

Reply:
Fine by me. If you do it before this goes into a real release, no one will even notice. Or probably no one will notice either way, as this is a brand new feature no one is using yet.

Well, Bun will notice ;)